### PR TITLE
Feat/migrate molot font

### PR DESCRIPTION
**Pull request by danielfpadilla**
#29 
Migrate Molot.otf from main branch and add global.css for global styling.

### CODE REVIEW

**Comment by jofftiquez** [comment]
Thanks @danielfpadilla will add a preview first before before further review thanks.

File: src/assets/css/global.css
Comment (jofftiquez) [inline src/assets/css/global.css:1]: @danielfpadilla please move this to `src/assets/css/tailwind.css` thanks

File: src/assets/css/global.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+body {
+  background-color: #fff;
+  color: #2c2c2c!important
```
Comment (jofftiquez) [inline src/assets/css/global.css:3]: Please add space between `#2c2c2c` and `!important`

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Thanks for this @danielfpadilla please see change requests

File: src/assets/css/global.css (hunk 1 above)
Comment (danielfpadilla) [reply src/assets/css/global.css]: Pushed the commit. thanks.

**Comment by jofftiquez** [comment]
Sorry for this, I was testing the deployment preview 🤣
image

File: src/assets/css/global.css
Comment (danielfpadilla) [reply src/assets/css/global.css:1]: @jofftiquez done bossing.

**Comment by danielfpadilla** [comment]
image
Add preview of the **ossph-font** css class to the website logo.

**Review by jofftiquez — APPROVED** [review]
Looks good, thank you!

**Pull request merged**